CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+Contribution Guide
+==================
+
+Discussing codestyle rules
+--------------------------
+
+If you want to discuss some of the codestyle rules, propose a new rule or suggest removing one, simply create a pull request or file an issue into the repository. Issues may be debating within one or two weeks, but in any case week after last comment in the discussion issue may be summarized by one of the maintainer of the repository.
+
+The issue may be closed with to outcomes:
```

Comment:
My formulation suggestions:
/may be closed with to outcomes:/will be closed with one of two possible resolutions:/